scripts/smoke_translate_docx.py

```
@@ -0,0 +1,100 @@
+    output_path = Path(args.output)
+    output_path.parent.mkdir(parents=True, exist_ok=True)
+
+    # Run translation
+    print("Running translation...")
+    cmd = [
+        'python', 'scripts/translate_docx.py',
+        '--in', str(input_path),
+        '--out', str(output_path),
+        '--batch', '1'  # Small batch for testing
+    ]
+    result = subprocess.run(cmd, capture_output=True, text=True, env=os.environ)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)